topics/dotenv/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: dot-env, env-file
+topic: dotenv
+display_name: .env
+short_description: .env is a file, used to store configuration values as key-value pairs, similar to the ini file format.
+related: ini, configuration-file, environment-variables
+---
+**.env** (**dotenv**) is a file, used to store configuration values as key-value pairs, similar to the ini file format. Since this file usually contains confidential information like credentials, its commonly prevented from being tracked by version control.
```

Comment:
```suggestion
**.env** (**dotenv**) is a file, used to store configuration values as key-value pairs, similar to the ini file format. Since this file usually contains confidential information like credentials, it's commonly prevented from being tracked by version control.
```

small grammar fix!

Reply:
Damn, should have seen it! 😅